### PR TITLE
Update dummy content to function as an introduction to the editor.

### PR DESCRIPTION
**Pull request by mtias**

### CODE REVIEW

**Comment by jasmussen** [comment]
I really quite like this. It's very inspiring to see it come together cohesively like this. Really makes me excited to polish the embed block, the image blocks, the rest of the post content... the whole thing. Very very cool.

There's one blocker, though. As a dane I consider it my solemn duty to defend the fact that LEGO is in allcaps. Always. `allCapsDangit()`!

File: post-content.js [hunk 1]
```diff
@@ -5,30 +5,58 @@ window._wpGutenbergPost = {
...
+			'<p>The goal of the new editor is to make adding rich content to WordPress simple and enjoyable. This whole post is composed of <em>pieces of content</em>—somewhat similar to Lego bricks—that you can move around and interact with. Move your cursor around and you\'ll notice the different blocks light up with outlines and arrows. Press the arrows to reposition blocks quickly, without fearing about losing things in the process of copying and pasting.</p>',
+			'<p>What you are reading now is a <strong>text</strong> <strong>block</strong>, the most basic block of all. A text block can have multiple paragraphs, if that\'s how you prefer to write your posts. But you can also split it by hitting enter twice. Once blocks are split they get their own controls to be moved freely around the post...</p>',
```
Comment (joyously) [inline post-content.js]: Why are the individual words marked bold, instead of one tag encompassing both words?

Comment (mtias) [inline post-content.js]: Ah, just an artifact, will fix.

**Comment by joyously** [comment]
Can you use a longer blockquote, with some `<em>` or bold or text alignment? A lot of themes make quotes italic or big or bold and it's nice to have those in there to see what it looks like.
I like to use this for testing themes:
`<blockquote cite="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote"><p>The <strong>HTML <code>&lt;blockquote&gt;</code> Element</strong> (or <em>HTML Block Quotation Element</em>) indicates that the enclosed text is an extended quotation. Usually, this is rendered visually by indentation (see <a href="https://developer.mozilla.org/en-US/docs/HTML/Element/blockquote#Notes">Notes</a> for how to change it). A URL for the source of the quotation may be given using the <strong>cite</strong> attribute, while a text representation of the source can be given using the <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/cite" title="The HTML Citation Element &lt;cite&gt; represents a reference to a creative work. It must include the title of a work or a URL reference, which may be in an abbreviated form according to the conventions used for the addition of citation metadata."><code>&lt;cite&gt;</code></a> element.</p></blockquote>
<cite>multiple contributors</cite> - MDN HTML element reference - blockquote`

**Comment by jasmussen** [comment]
Added some images and text at the bottom. Should we find a different video, or move it upwards?

**Comment by jasmussen** [comment]
I moved the video and added a caption. I think we should merge this in and keep moving.

**Comment by mtias** [comment]
Merged, we'll keep refining this.

**Comment by aduth** [comment]
The embed block's comment syntax should have been updated with consideration for #626.

**Comment by mtias** [comment]
I missed it was broken after the rebase.

**Comment by jasmussen** [comment]
Pretty sure that was my fault. Apologies, fixing.

**Comment by mtias** [comment]
@jasmussen already submitted #675

**Comment by jasmussen** [comment]
Hah, sorry, deleting mine.

**Pull request merged**